zebra/zebra_vty.c

```diff
@@ -192,6 +192,14 @@ static char re_status_output_char(const struct route_entry *re,
 				star_p = true;
 		}
 
+		if (zrouter.asic_offloaded
+		    && CHECK_FLAG(re->flags, ZEBRA_FLAG_TRAPPED))
+			return 't';
+
+		if (zrouter.asic_offloaded
+		    && !CHECK_FLAG(re->flags, ZEBRA_FLAG_OFFLOADED))
+			return 'o';
+
```

Comment:
will the change here lead to a loss of stars ("*") for a lot of routes - I guess I'm assuming that many routes will get the OFFLOAD flag set, when the feature is available end-to-end? maybe we need to consider expanding this part of the display beyond a single character, so it's capable of conveying more info?

Reply:
please reread.  I only change the sense if the offload *failed*